composer/optim/decoupled_weight_decay.py

```diff
@@ -193,18 +193,10 @@ class DecoupledAdamW(AdamW):
             lambda param, optim_state, step_tensor: torch.linalg.vector_norm(optim_state['exp_avg']),
         'l2_norm/param':
             lambda param, optim_state, step_tensor: torch.linalg.vector_norm(param.data),
-        'l2_norm/second_moment_sqrt':
```

Comment:
Is this change part of throttling optimizer monitor or should it be a separate pr that changes weight decay logic?

Reply:
This removes logging for some optimizer monitor features that are no longer necessary per @bmosaicml's feedback. It shouldn't change any functional logic, just remove logging :)